kong/api/routes/cache.lua

```diff
@@ -1,18 +1,60 @@
 local responses = require "kong.tools.responses"
 local cache = require "kong.tools.database_cache"
 
+local cache_names = {}
```

Comment:
This doesn't need to be in the plugin scope. How about enclosing it with `do ... end`?
```lua
local caches

do
  local cache_names = {}
  for name in pairs(caches) do 
    cache_names[#cache_names + 1] = "'" .. name .. "'" 
  end
  
  -- make order deterministic, for test purposes
  table.sort(cache_names)

  cache_names = table.concat(cache_names, ", ")

  caches = setmetatable({
    ...
  })
end
```